theregister.com.txt

```diff
@@ -0,0 +1 @@
+theregister.co.uk.txt
```

Comment:
@fivefilters is it working? 🤔

Reply:
Sadly GitHub is not really user-friendly when talking about symlinks.

This file is a symlink (_see `new file mode 120000` in the patch_), it works on Graby. It should also work for @fivefilters

Reply:
Hmm, I don't think we have any symlinked files in this repo. It would be a nice way to reduce duplication, but I'll need to test this some more. A little concerned that it will create difficulties for Windows users. I'll also need to check to see if the symlinks get preserved properly in the zip archives that Github produces (Full-Text RSS uses those zip files for updating extraction rules).

So for now I'd suggest we duplicate the files with a comment that the contents should mirror the contents of the other file - that's how I've been doing it for similar sites up to now.